cli-readme/README.md

```diff
@@ -67,10 +65,10 @@ plural crypto unlock
 
 ## Destroying Your Cluster
 
-You can tear down the cluster at any time using `plural destroy`.  If it gets jammed on a specific app, you can simply rerun with `plural destroy --from APP`.
+You can tear down the cluster at any time using `plural destroy`.  If it gets jammed on a specific app, you can simply rerun with `plural destroy --from APP_NAME`.
 
-In break glass scenarios where destroy doesn't want to behave, the bulk of the cloud resources are in the bootstrap app, and you can simply run `cd bootstrap && terraform destroy` manually.  If you encounter this feel free to let us know so we can improve our packaging
+In break glass scenarios where destroy doesn't want to behave, the bulk of the cloud resources are in the bootstrap app, and you can simply run `cd bootstrap && terraform destroy` manually.  If you encounter this feel free to let us know so we can improve our packaging.
 
 ## Support
 
-If you have any issues with plural or any of the apps in the catalog, we pride ourselves on providing prompt support.  You can join our discord at https://discord.com/invite/pluralsh and we'll try to troubleshoot the issue for you.  You can find common troubleshooting tips [here](https://docs.plural.sh/reference/troubleshooting) as well
\ No newline at end of file
+If you have any issues with plural or any of the apps in the catalog, we pride ourselves on providing prompt support.  You can join our Discord at [https://discord.gg/pluralsh](https://discord.gg/pluralsh) and we'll try to troubleshoot the issue for you.  You can find common troubleshooting tips [here](https://docs.plural.sh/reference/troubleshooting) as well.
```

Comment:
Doesn't markdown auto-linkify?

Reply:
oh true good point, I guess this doesn't hurt though